samples/highcharts/accessibility/internationalization/demo.html

```diff
@@ -0,0 +1,27 @@
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/stock/modules/data.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+<link rel="stylesheet" type="text/css" href="//github.com/downloads/lafeber/world-flags-sprite/flags32.css" />
+
+<div role="radiogroup" style="min-width: 310px; max-width: 600px; margin: 0 auto">
+    <div role="radio" aria-checked="true" tabindex="0">
```

Comment:
The `radiogroup` and `radio` roles are really only needed for custom radio widgets, where you are simulating radio buttons using e.g. `<div>`s. Take a look at this example for a better approach in this case using the `<fieldset>` element: https://www.w3.org/WAI/tutorials/forms/grouping/ 

Note that each flag should have an accessible name, perhaps using `aria-label`, or maybe the span can contain text as well? As it is right now, the screen reader user can't tell which language is being selected.